src/Microsoft.ServiceFabric.Actors/Runtime/ActorStateProviderHelper.cs

```diff
@@ -295,7 +295,7 @@ internal static IActorStateProvider GetActorStateProviderOverride()
                 var stateProviderOverrideSectionName = ActorNameFormat.GetActorStateProviderOverrideSectionName();
                 var attributeTypeKey = ActorNameFormat.GetActorStateProviderOverrideKeyName();
 
-                // Load the ActorStateProviderAttribute Type from the Configuration settings
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.